src/components/Dissolution/CompleteResolution.vue

```diff
@@ -605,6 +607,7 @@ export default class CompleteResolution extends Mixins(CommonMixin, DateMixin, E
   }
 
   async mounted (): Promise<void> {
+    // wait for components to load/stabilize then update validation state in store
```

Comment:
I saw a lot of `await Vue.nextTick()` in this component, suggesting there is some timing fragility. This may be what we're seeing with the stepper saying the component in incomplete, but on the step itself everything looks OK.